internal/validation/validation.go

```diff
@@ -31,6 +31,14 @@ func ValidateUserObjectRelation(typesys *typesystem.TypeSystem, tk *openfgav1.Tu
 	return nil
 }
 
+func ValidateUserObjectRelationCondition(typesys *typesystem.TypeSystem, tk *openfgav1.TupleKey) error {
```

Comment:
can we add a comment to this new function?

Reply:
👍